src/index.ts

```diff
@@ -24,6 +24,39 @@ type LooseNode = Node & {
   value: any;
 };
 
+const generateHeaders = async () => {
+  await writeFile(
+    ".vercel/output/static/_headers",
+    `\n/_next/static/*\n  Cache-Control: public,max-age=31536000,immutable\n`,
+    {
+      // in case someone configured redirects already, append to the end
+      flag: "a",
+    }
+  );
+};
+
+const generateRoutes = async () => {
+  try {
+    await writeFile(
+      ".vercel/output/static/_routes.json",
+      JSON.stringify({
+        version: 1,
```

Comment:
We also accept a `description` property, which [Wrangler](https://github.com/cloudflare/workers-sdk/blob/f61de1dd1b09cb45fefb6e2ace2e615f6a87ef4b/packages/wrangler/src/pages/constants.ts#L18) and [SvelteKit](https://github.com/sveltejs/kit/blob/c45c35e7b40837cd1bea3fc26cfb831478e03c00/packages/adapter-cloudflare/index.js#L126) add a small note to describe who built this file. We could add something like "Built with @cloudflare/next-on-pages@0.0.3."

Reply:
Added.